src/commctl/client_script.py

```diff
@@ -255,7 +255,7 @@ def create_host(self, address, **kwargs):
         infile = kwargs['ssh-priv-key']
         b64_bytes = base64.b64encode(infile.read())
         data['ssh_priv_key'] = b64_bytes.decode()
-        if 'cluster' in kwargs:
+        if kwargs['cluster'] is not None:
```

Comment:
Heh, that's what I thought as well. Turns out if it's not provided, it will be `None`.